src/ianitor/args_parser.py

```diff
@@ -143,6 +143,7 @@ def get_parser():
 
     parser.add_argument(
         "--port",
+        type=int, default=0,
```

Comment:
perhaps there is no default value here, argument `--port` still equals to `0` even leave it empty.